events/wc_dark_cult_events.txt

```diff
@@ -1114,6 +1170,235 @@ narrative_event = {
 				character_event = { id = MNM.10030 }
 			}
 			narrative_event = { id = WCDAC.5050 }
+			character_event = { id = WCDAC.5041 days = 7 }
+			character_event = { id = WCDAC.5045 years = 5 }
+		}
+	}
+}
+character_event = {
+	id = WCDAC.5041
+	desc = {
+		trigger = { NOT = { has_character_flag = hunted_by_garona } }
+		text = EVTDESC_WCDAC_5041_first_kill
+	}
+	desc = {
+		trigger = {
+			trigger = { has_character_flag = hunted_by_garona }
+		}
+		text = EVTDESC_WCDAC_5041_later_kills
+	}
+	picture = GFX_evt_ahnqiraj
+	border = GFX_event_normal_frame_intrigue
+
+	is_triggered_only = yes
+	
+	trigger = {
+		NOT = { trait = travelling }
+		c_10620 = { is_alive = yes }
+		character = 52000
+	}
+	
+	immediate = {
+		random_courtier = {
+			limit = {
+				NOT = { character = 10620 }
+				religion = old_gods_worship
+			}
+			death = { death_reason = death_murder }
+		}
```

Comment:
I hope it's not the final version because here's a small list of characters Garona able to kill:
- Children
- Dragons
- C'Thraxxi
- Characters being away like Garona herself
- Incapable

Reply:
"Characters being away like Garona herself" i'll fix this but i dont see the problem with the rest

Reply:
> "Characters being away like Garona herself" i'll fix this but i dont see the problem with the rest

So, Garona is able to kill C'Thraxxi? And don't mind killing children?

Reply:
well she can kill chars that are as strong as c'thraxxi with this event

Reply:
Sejtalk:
Combat rating needs rebalancing.

Also sejtalk
> well she can kill chars that are as strong as c'thraxxi with this event

Reply:
well it's more of a plot device

Reply:
> well it's more of a plot device

Though, the event not takes it into consideration.

Plus, the event mentions it was Garona who killed them, so it looks like Garona is the most powerful entity on Azeroth.

Reply:
you can plot to kill them so it's fair game imo